src/internal.c

```diff
@@ -9451,6 +9457,22 @@ static int GetRecordHeader(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
             break;
         case no_type:
         default:
+#ifdef OPENSSL_ALL
+            {
+                char *method = (char*)input + start;
```

Comment:
Couldn't this read past the buffer by 5 characters? Should we also check `size`? This seems like an odd case to require..., since it will fail later. Should this be `#if defined(OPENSSL_ALL) || defined(WOLFSSL_APACHE_HTTPD)`?

Reply:
This will read at most 5 bytes which is the amount `memcpy`'d at the beginning of the function. Checking `*size` doesn't make sense as it is set from the assumed "record header" (would be set to one of the ASCII bytes). The apache config defines `OPENSSL_ALL` so no need to check against it.